src/options.ts

```diff
@@ -11,6 +12,8 @@ export interface TerminalScreenshotOptions {
   readonly backgroundColor: string;
   // Type of the screenshot to be generated. (default: png)
```

Comment:
related to the other comments, I think it is an opportunity to remove `backgroundColor` and `margin` options, and rely on the theme config. WDYT?

Reply:
superseded by https://github.com/OmarTawfik/terminal-screenshot/pull/11#discussion_r1492581182